conf/formTemplateSchema.json

```diff
@@ -729,7 +729,215 @@
             "$ref": "#/$defs/fields"
           }
         }
-      }
+      },
+      "allOf": [
+        {
+          "anyOf": [
+            {
+              "not": {"required": ["infoText"]}
```

Comment:
Would it be possible to format this? (applies to whole json)

My formatter do this for example:
```
              "not": {
                "required": [
                  "infoText"
                ]
              }
```